### PR TITLE
fix bazel build

### PR DESCRIPTION
**Pull request by zhexuany**
Fix bazel build in this repo.

### CODE REVIEW

File: src/main/java/org/tikv/BUILD [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 java_library(
-    name = "tikv-java-client-lib",
+    name = "tikv_java_client_lib",
     srcs = glob(
         ["**/*.java"],
     ),
     deps = [
-        "//:java",
+        "//:tikv_proto_java_lib",
+#        ":tikv_parser_lib",
```
Comment (zhexuany) [inline src/main/java/org/tikv/BUILD]: comment for now since soon or later we will merge parser code into this repo.

**Comment by birdstorm** [comment]
Can you please add some documents for `bazel`? I observe the usage of `bazelisk` rather than `bazel` in your code. @zhexuany

**Comment by zhexuany** [comment]
@birdstorm PTAL

**Comment by birdstorm** [comment]
Closed since we are not going to support bazel.

**Pull request closed without merging**